src/configs/nuage/elasticsearch/tabification/index.js

```diff
@@ -10,3 +10,4 @@ export { default as defaultTabify } from './defaultTabify';
 export { default as areaHeatmap } from './areaHeatmap';
 export { default as aqlCoverage } from './aqlCoverage';
 export { default as aqlPassFail } from './aqlPassFail';
+export { default as expressHeatmap } from './expressHeatmap';

```

Comment:
What is the delta between expressHeatmap and areaHeatmap? Do we need this much of a duplication of code?

Reply:
This is a correct comment, in fact all the tabification files need to be trimmed down more. Can I do it via a separate PR and leave the custom function intact in all these?

Reply:
Ok